### ISSUE TITLE
Unreasonable prediction on test set with resnet_v2 for specific batch size

### ISSUE THREAD

**Issue by wooginawunan** [tensorflow/tensorflow#17421]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:  Ubuntu 16.04.3 LTS
- **TensorFlow installed from (source or binary)**: Binary from https://storage.googleapis.com/tensorflow/linux/gpu/tensorflow-0.9.0-cp35-cp35m-linux_x86_64.whl
- **TensorFlow version (use command below)**: 1.3.0
- **Python version**: 3.5.1+
- **Bazel version (if compiling from source)**: /
- **GCC/Compiler version (if compiling from source)**: / 
- **CUDA/cuDNN version**: V7.5.17
- **GPU model and memory**: Tesla M40 12G
- **Exact command to reproduce**: /

### Describe the problem
I am using resnet_v2 provided in slim to train a classifier. However, there is an unexpected behaviour when I tried to get the predictions on test set and I am sure that I turned off the training mode by pass is_training as False. 

My model returned unreasonable prediction scores on specific testing mini batch size of 650 and 649. 
I randomly checked several testing batch sizes (from 50 - 850, which my GPU can afford ), their predictions on same input data were all consistent with each other, except for size 650 and 649.  Actually even two identical patches in the mini batch had different scores when I am using batch size 650 and 649.  To verify the issue, I also tried to train the model with different input data and different training mini batch size. However, the problem is still there with the same weird mini batch size 650 and 649 :(.

**Comment by shivaniag** [comment]
@derekjchow: can you take a look?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 61 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
